src/Sulu/Bundle/LocationBundle/Geolocator/GeolocatorLocation.php

```diff
@@ -33,6 +33,13 @@ class GeolocatorLocation
      */
     protected $displayTitle;
 
+    /**
+     * Name to display.
+     *
+     * @var string
+     */
+    protected $displayName;
```

Comment:
the error appears in `setDisplayName` a public method which access this undefined attribute the function is also called in `setName`. I'm also not sure if it should just call `setDisplayTitle` instead 😕